sale_lead_time/models/lead_time_profile.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2025 Quartile
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class LeadTimeProfile(models.Model):
+    _name = "lead.time.profile"
```

Comment:
Shouldn't the lead time profile externalized as a base module ?

Reply:
I think it is better to have a base module even we don't have use case for other models at the moment.